packages/sheets-numfmt/src/controllers/numfmt.menu.controller.ts

```diff
@@ -29,6 +31,8 @@ export const DefaultSheetNumfmtConfig = {};
 
 @OnLifecycle(LifecycleStages.Rendered, NumfmtMenuController)
 export class NumfmtMenuController extends Disposable {
+    private _currencySymbol$ = new BehaviorSubject<keyof typeof currencyMap>('ZH_CN');
```

Comment:
It is better to infer the default currency symbol for the browser's `navgiator` info. Since this property is at UI layer, it is considered as harmless.

Reply:
It it recommended to make property `readonly` whenever possible.